pandas/core/frame.py

```diff
@@ -9142,6 +9142,23 @@ def _gotitem(
     2    8.0
     3    NaN
     dtype: float64
+
+    Aggregate functions on DataFrames with mixed non-numeric and numeric columns.
+
+    >>> df = pd.DataFrame([['a', 1, 4],
+    ...                    ['b', 2, 5],
+    ...                    ['c', 3, 6]],
+    ...                   columns=['A', 'B', 'C'])
+
+    Works equivalently as above. Add argument `numeric_only=True` to avoid
+    exceptions or warnings.
```

Comment:
Nit: The warning will go away soon (in 2.0) so I am not sure it's worth including here